ros_buildfarm/templates/ci/create_workspace.Dockerfile.em

```diff
@@ -69,6 +69,10 @@ RUN echo "@today_str"
     dependency_versions=dependency_versions,
 ))@
 
+# needed for 'vcs custom --git --args merge' invocation
+RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q --no-install-recommends sudo
+RUN sudo -H -u buildfarm -- git config --global user.email "jenkins@@ros.invalid" && sudo -H -u buildfarm -- git config --global user.name "Jenkins ROS"
```

Comment:
Why is this done using `sudo` instead of just putting it after the `USER` entry on line 77?

Reply:
I just did the same as in the ROS 2 CI repo: https://github.com/ros2/ci/blob/573f0dedc9cc2d6b14380ff67a2177fd3701ddbd/linux_docker_resources/Dockerfile#L177